static/js/compose.js

```diff
@@ -530,17 +573,23 @@ exports.transmit_message = function (request, success, error) {
     }
 };
 
+function report_content_not_sent(request) {
+    var message_undelivered_id = "" + request.local_id;
+    message_undelivered_id = "div[zid='" + message_undelivered_id.replace( /(:|\.|\[|\]|,|=|@)/g, "\\$1" ) + "']";
+    var message_undelivered_tag = $(message_undelivered_id).find(".message_content");
+    message_undelivered_tag.after("<div class='message_undelivered' style='color: red;'>Your message was not sent</div>");
```

Comment:
Is there UI to remove the message from the feed once the users sees that it has timed out?